Description: pom.xml, change packaging type from bundle to jar.
 Since this library is a dependency of maven-bundle-plugin
 we should use default "jar" packaging type to allow a first bootstrap build.
Forwarded: not-needed
Author: Damien Raude-Morvan <drazzib@debian.org>
Last-Update: 2011-05-11

--- a/pom.xml
+++ b/pom.xml
@@ -24,7 +24,7 @@
     <relativePath>../pom/pom.xml</relativePath>
   </parent>
   <modelVersion>4.0.0</modelVersion>
-  <packaging>bundle</packaging>
+  <packaging>jar</packaging>
   <name>Apache Felix Main</name>
   <artifactId>org.apache.felix.main</artifactId>
   <version>2.0.5</version>
